### PR TITLE
[HOTFIX] Pascalvoc importer should not fail on float values

### PR DESCRIPTION
**Pull request by andreaazzini**
Addresses https://github.com/v7labs/darwin-py/issues/137

### CODE REVIEW

File: darwin/importer/formats/pascalvoc.py [hunk 1]
```diff
@@ -1,28 +1,46 @@
 import xml.etree.ElementTree as ET
 from pathlib import Path
-from typing import Optional
+from typing import List, NoReturn, Optional, Union
 
 import darwin.datatypes as dt
 
 
-def parse_file(path: Path) -> Optional[dt.AnnotationFile]:
+def parse_file(path: Path) -> Union[Optional[dt.AnnotationFile], NoReturn]:
```
Comment (Fl4m3Ph03n1x) [inline darwin/importer/formats/pascalvoc.py:8]: Equivalent to: 
```
Union[dt.AnnotationFile, None, NoReturn]
```

File: darwin/importer/formats/pascalvoc.py [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
-    ymax = int(bndbox.find("ymax").text)
+    bndbox = _find_element(annotation_object, "bndbox")
+    xmin = int(float(_find_text_value(bndbox, "xmin")))
+    xmax = int(float(_find_text_value(bndbox, "xmax")))
+    ymin = int(float(_find_text_value(bndbox, "ymin")))
+    ymax = int(float(_find_text_value(bndbox, "ymax")))
 
     return dt.make_bounding_box(class_name, xmin, ymin, xmax - xmin, ymax - ymin)
+
+
```
Comment (Fl4m3Ph03n1x) [inline darwin/importer/formats/pascalvoc.py:34]: Should we document these underscore functions as private?

File: darwin/importer/importer.py [hunk 3]
```diff
@@ -21,7 +25,7 @@ def build_main_annotations_lookup_table(annotation_classes):
 def find_and_parse(
     importer: Callable[[Path], Union[List[dt.AnnotationFile], dt.AnnotationFile, None]],
     file_paths: List[Union[str, Path]],
-) -> (List[dt.AnnotationFile], List[dt.AnnotationFile]):
+) -> Tuple[List[dt.AnnotationFile], List[dt.AnnotationFile]]:
     # TODO: this could be done in parallel
```
Comment (Fl4m3Ph03n1x) [inline darwin/importer/importer.py:29]: Should we create tickets for TODO tags?
How can we prevent this codebase from being filled with TODO/FIXME tags? How are we addressing them?

**Review by Fl4m3Ph03n1x — COMMENTED** [review]
Nice tests!
Some minor questions and a suggestion for a change.

File: darwin/importer/importer.py (hunk 3 above)
Comment (andreaazzini) [reply darwin/importer/importer.py:29]: Good point, done: https://linear.app/v7labs/issue/V7-1221/parsing-files-in-importer-can-be-done-in-parallel

**Review by Fl4m3Ph03n1x — APPROVED** [review]

**Comment by Fl4m3Ph03n1x** [comment]
Approved !

**Pull request merged**